examples/motoko_examples/whoami/test/package.json

```diff
@@ -4,11 +4,11 @@
         "test": "cd .. && KYBRA_REBUILD=true ts-node --transpile-only --ignore=false test/test.ts"
     },
     "dependencies": {
-        "@dfinity/identity": "0.14.1",
-        "azle": "github:demergent-labs/azle"
+        "@dfinity/identity": "0.14.1"
```

Comment:
I'm confused at when a test dependency should be in dependencies vs dev dependencies